mimer-config/src/main/java/com/avanza/astrix/config/MapConfigSource.java

```diff
@@ -31,6 +31,15 @@
 	
 	private final ConcurrentMap<String, ListenableStringProperty> propertyValues = new ConcurrentHashMap<>();
 	
+	public MapConfigSource() {
+	}
+
+	public static MapConfigSource of(Map<String, ? extends Object> source) {
+		MapConfigSource configSource = new MapConfigSource();
+		source.entrySet().forEach(e ->configSource.set(e.getKey(), e.getValue().toString()));
+		return configSource;
+	}
```

Comment:
Man kan göra `forEach` direkt på `Map`:
```suggestion
	public static MapConfigSource of(Map<String, ?> source) {
		MapConfigSource configSource = new MapConfigSource();
		source.forEach((key, value) -> configSource.set(key, value.toString()));
		return configSource;
	}
```
(`? extends Object` är redundant också, det räcker med `?`)